docs/migrating/common_issues.md

```diff
@@ -15,39 +15,19 @@ issues that can occur after migration to eZ Platform.
 
     Remember about [**proper backup**](../guide/backup.md) before running any of the commands below.
 
-## Regenerating URL Aliases
+## Regenerating URL aliases
 
-Basic `ezplatform:regenerate:` command implements the cleanup script that will check if 
-Fields of given Field Type have correct sort key, and update it if needed. 
+To regenerate URL aliases, use the `ezplatform:urls:regenerate-aliases` command.
+See [Regenerating URL aliases](../guide/url_management.md#regenerating-url-aliases) for more information.
 
-For this specific issue use:
+!!! note
 
-```
-php bin/console ezplatform:regenerate:legacy_storage_url_aliases
-```
-
-It will regenerate URL aliases for Locations and migrate existing custom Location and 
-global URL aliases to a separate database table. The separate table must be named 
-`__migration_ezurlalias_ml` and should be created manually to be identical (but empty) 
-as the existing table `ezurlalias_ml` before the command is executed.
-
-After the script finishes, to complete migration the table should be renamed to 
-`ezurlalias_ml` manually. Using available options for `action` argument, you can back up 
-custom Location and global URL aliases separately and inspect them before restoring them 
-to the migration table. They will be stored in backup tables named 
-`__migration_backup_custom_alias` and `__migration_backup_global_alias` (created 
-automatically).
-
-It is also possible to skip custom Location and global URL aliases altogether and 
-regenerate only automatically created URL aliases for Locations (use the `autogenerate` 
-action to achieve this). During the script execution the database should not be modified. 
-Since this script can potentially run for a very long time, to avoid memory exhaustion run 
-it in production environment using the `--env=prod` switch.
+    This command replaces the deprecated `ezplatform:regenerate:legacy_storage_url_aliases command.`
```

Comment:
I know you don't like to repeat the doc, but maybe we could also put here info that the reason for deprecating is that the old command didn't keep history? (could be rephrased).